### PR TITLE
A refactor into a separate single script file

### PR DESCRIPTION
**Pull request by RedundantCich**
I've done a couple changes to make extending the scripts easier:
- separated the beginning of each file into a single separate script that is used by all others
- made iflag arguments into a global inside the singular script file
- changed rom ids to a global variable

Take a look if you're OK with those changes.
If not, it's cool.
If you would prefer to modify something, let me know in the comments to the PR.

### CODE REVIEW

**Comment by Lx32** [comment]
I think that is a good idea, I'll try to improve this creating a general parametric script. Maybe I'll try to use python.

**Comment by Lx32** [comment]
There is an error on 3 Count Bout graphics files. If I have time I'll try to check better. The CRC doesn't match.

**Comment by RedundantCich** [comment]
> There is an error on 3 Count Bout graphics files. If I have time I'll try to check better. The CRC doesn't match.

You can use a tool i linked in the comments to convert these C files. Probably worth analysing how they are converted and moving the code to our repo

**Comment by RedundantCich** [comment]
> There is an error on 3 Count Bout graphics files. If I have time I'll try to check better. The CRC doesn't match.

Turns out, it was an easy fix. I've update the file, and it just works now :)
Thanks for the comment

File: goTH [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+checkForZip
+checkForCompiledSwizzle
+cleanFilesForReRun $MY_OUT_ROM_DIR $MY_ZIP_NAME
+
+# Check if s2.bin is available, and copy it as ROM_ID-s1.s1.
+if [ ! -f "$MY_RAW_ROMS_DIR"/s2.bin ]; then
+    echo "File s2.bin not found in $MY_RAW_ROMS_DIR!"
+fi
+cp "$MY_RAW_ROMS_DIR"/s2.bin "$MY_OUT_ROM_DIR"/"$ROM_ID"-s1.s1
+
```
Comment (lioneltrs) [inline goTH:32]: Could we add a `chmod -x  "$MY_OUT_ROM_DIR"/"$ROM_ID"-s1.s1`

Comment (RedundantCich) [reply goTH:32]: Sure, I'll add it today after work.

Comment (RedundantCich) [reply goTH:32]: Ah, it's probably best to move s1.bin creation to a separate method too.
I'll do it over the weekend then.

**Comment by zilladotexe** [comment]
Nice work all. Looks like that might be the last of the games, at least for now.

**Pull request merged**